pkg/cloud/azure/actuators/machine/actuator.go

```diff
@@ -125,19 +159,21 @@ func (a *Actuator) Update(ctx context.Context, cluster *clusterv1.Cluster, machi
 		CoreClient: a.coreClient,
 	})
 	if err != nil {
-		return errors.Errorf("failed to create scope: %+v", err)
+		return a.handleMachineError(machine, apierrors.UpdateMachine("failed to create scope: %v", err), updateEventAction)
```

Comment:
Message should be `failed to update`?

Reply:
Machine scope is always created, never updated. So, it's correct to use `failed to create scope`.